src/components/core/geoChart/GeoChartLegendRenderer.tsx

```diff
@@ -59,3 +54,31 @@ function renderPushpinSizeLegend(sizeValues: number[], format: string, locale: s
         </IntlWrapper>
     );
 }
+
+function renderPushpinSegmentByLegend(props: IGeoChartLegendRendererProps): JSX.Element {
```

Comment:
It could be `renderPushpinCategoryLegend `

Reply:
fixed